src/modules/zk-evm-bridge/l1-bridge/utils.ts

```diff
@@ -21,32 +22,33 @@ import { astarNativeTokenErcAddr } from 'src/modules/xcm';
 type MerkleProof = {
   main_exit_root: string;
   merkle_proof: string[];
-  rollup_merkle_proof: string[];
+  // Todo: remove '?'
+  rollup_merkle_proof?: string[];
```

Comment:
@impelcrypto what should occur so that you can remove the ?

Reply:
This key doesn't exist in zKatana's API but zKyoto and mainnet. So we should remove `?` when we migrate the testnet to zKyoto